lms/djangoapps/bulk_email/tasks.py

```diff
@@ -375,7 +376,12 @@ def _get_source_address(course_id, course_title, truncate=True):
     # character appears.
     course_name = re.sub(r"[^\w.-]", '_', course_id.course)
 
-    from_addr_format = u'"{course_title}" Course Staff <{course_name}-{from_email}>'
+    with override_language(settings.LANGUAGE_CODE):
```

Comment:
Why do you need to use a context manager here? Is it a matter of changing the language to whatever is set as the base language for the openedx language vs. the user's language preference? I'm trying to understand why emails for all courses should be in the same language.

A base language can now be set per course in Studio's course settings. Would it be better to use that value?

![image](https://cloud.githubusercontent.com/assets/484484/17223173/72bf6792-54c9-11e6-8290-01be2f8a6f55.png)